idea/idea-frontend-fir/src/org/jetbrains/kotlin/idea/frontend/api/fir/diagnostics/KtFirDiagnostics.kt

```diff
@@ -534,7 +539,7 @@ sealed class KtFirDiagnostic<PSI: PsiElement> : KtDiagnosticWithPsi<PSI> {
         override val diagnosticClass get() = NonFinalMemberInObject::class
     }
 
-    abstract class ManyCompanionObjects : KtFirDiagnostic<PsiElement>() {
+    abstract class ManyCompanionObjects : KtFirDiagnostic<KtObjectDeclaration>() {
```

Comment:
Why is this change necessary?

Reply:
What is wrong bro?

Reply:
Hmm. Weird. I actually didn't change this and this is changed in https://github.com/JetBrains/kotlin/commit/6fee46316eb9836a13a8367c001d9122fff5974f. Let me sync master again and re-push.

Reply:
Huh, ok I guess the generator was not run then. 🤷